healthd/healthd_mode_charger.cpp

```diff
@@ -301,6 +301,23 @@ void Charger::BlankSecScreen() {
     }
 }
 
+void Charger::UpdateLedState() {
```

Comment:
I don't really like commiting WIP code. Maybe remove the led state part for the moment?